src/components/views/rooms/MessageComposer.tsx

```diff
@@ -372,7 +372,7 @@ export class MessageComposer extends React.Component<IProps, IState> {
         const { isRichTextEnabled, composerContent } = this.state;
         const convertedContent = isRichTextEnabled
             ? await richToPlain(composerContent)
-            : await plainToRich(composerContent);
+            : await plainToRich(composerContent, false);
```

Comment:
ok, so what's the difference between ` plainToRich(foo, false)` and `plainToRich(foo, true)`?

I got as far down the rabbithole as https://github.com/matrix-org/matrix-rich-text-editor/blob/61cdc442861a6b9401260fc36f68a4f58e6b2803/platforms/web/lib/conversion.ts#L76 but lost the will to dig any deeper...

Reply:
There are differences between the html required for correct display in the composer vs the html required for the content of a message. We can use the rust model to output either, this second argument allows us to choose whether the html we get back is in message format (`true`) or in "composer format" (`false`). 

I'll add some documentation to the function in the RTE repo, which may require some duplication in this repo for the time being (due to the way the dynamic imports are handled for these functions).

Reply:
Documentation comments have been added to the dynamic imports in https://github.com/matrix-org/matrix-react-sdk/blob/84880d7180f9d50fea34d52195e2be131302452e/src/components/views/rooms/wysiwyg_composer/DynamicImportWysiwygComposer.tsx#L51-L58

Reply:
very helpful, thanks!